src/arm/bbb-bone-buses.dtsi

```diff
@@ -0,0 +1,474 @@
+/*
+ * Copyright (C) 2020 Deepak Khatri <deepaklorkhatri7@gmail.com>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ */
+
+#include <dt-bindings/board/am335x-bbb-pins.h>
+
+/********/
+/* LEDs */
+/********/
+&{/} {
+	leds {
+		pinctrl-names = "default";
+		compatible = "gpio-leds";
+
+		led_P8_03 {
+			gpios = <&gpio1 6 0>;
+			status = "disabled";
```

Comment:
Ohh this, I wanted to ask this to you. both "reserved" and "disabled" worked for me which should we choose?

From Deice tree specifications I found this:

```
"disabled" 
Indicates that the device is not presently operational, but it might become operational in the future
(for example, something is not plugged in, or switched off).
Refer to the device binding for details on what disabled means for a given device.

"reserved"
Indicates that the device is operational, but should not be used. Typically this is used for devices that
are controlled by another software component, such as platform firmware.
```

Reply:
I like the choice of "disabled".